nwb_conversion_tools/datainterfaces/blackrockdatainterface.py

```diff
@@ -13,6 +13,7 @@
 from .interface_utils.brpylib import NsxFile
 
 PathType = Union[str, Path, None]
+OptionalPathType = Optional[Union[str, Path]]
```

Comment:
Both these lines are redundant (`Optiona[x] = Union[x, None]`). Use `PathType=Union[str, Path]` for non-optional path annotations and `OptionalPathType = Optional[PathType]` for non-required ones.

Reply:
I'm not sure what's your suggestion here

Reply:
I think Cody was suggesting to put `PathType` in the brackets of `Optional`, and to remove `None` from `PathType = ...`. 
So part of the second line is actually redundant then. I didn't test this or know what this code is about to be honest, just interpreting while browsing through. Maybe there is a good reason for the `None` in the first line and it should not be in the second?

So:
```
PathType = Union[str, Path]
OptionalPathType = Optional[PathType]
```